LEDCylce_randomModeSwitch

```diff
@@ -1,4 +1,4 @@
-// Include NeoPixel library
+// Include NeoPixel library!
```

Comment:
I need you to diff the whole file for me to comment on all of the code.

Easiest way to do that is a file rename probably

Reply:
I changed the name and added a change at the end of the code thinking it would unlock all of it for you to view, not sure if it has or now